src/components/Footer.tsx

```diff
@@ -10,7 +10,19 @@ import {
 export const Footer = () => {
   return (
     <Wrapper>
-      <h5>Made by Satellytes&nbsp; · &nbsp;</h5>
+      <h5>
```

Comment:
My original idea was actually just to link the imprint to the existing "Made by Satellytes" text, but this way is much better, of course.